nano/secure/blockstore_partial.hpp

```diff
@@ -27,15 +23,35 @@ namespace nano
 template <typename Val, typename Derived_Store>
 class block_predecessor_set;
 
+template <typename Val, typename Derived_Store>
```

Comment:
Moved the macro to a friend function declared outside class block_store_partial.